go/libkb/rpc_exim.go

```diff
@@ -869,7 +869,7 @@ func (ckf ComputedKeyFamily) exportPublicKey(key GenericKey) (pk keybase1.Public
 			i++
 		}
 		pk.PGPIdentities = ids
-		pk.IsRevoked = len(pgpBundle.Revocations) > 0
+		pk.IsRevoked = len(pgpBundle.Revocations)+len(pgpBundle.UnverifiedRevocations) > 0
```

Comment:
This marks 3rd party revoked key as `[Revoked]` in PGP list. I'm not sure what other impact this will have.